src/lib/components/Title.svelte

```diff
@@ -0,0 +1,15 @@
+<h4><slot /></h4>
+
+<style lang="scss">
+  @use "../styles/mixins/text";
+
+  h4 {
+    margin: 0;
+    color: white;
```

Comment:
This is where having value/reference tokens would be handy.

Reply:
yes definitely!

note: style is the same as before except a grid position attribute which was actually unused
it disappeared because the component was removed in nns-dapp. I copied the style from history, no other change.